### DISCUSSION TITLE
ExprRandom | Removal and replacement

### DISCUSSION THREAD

**Discussion by Fusezion** [SkriptLang/Skript#5254] [Suggestions]
### Suggestion

- [ ] Remove `ExprRandom` from skript's expressions
- [ ] Add a new expression class called `ExprClassinfo`
- [ ] ExprClassinfo: support for `random %*type%` pattern
- [ ] ExprClassinfo: `first/last [%-number%] %*type%` pattern
- [ ] ExprClassinfo:  ``%number%(st|nd|rd|th) %*type%`` pattern

Removal of ExprRandom brings room for a more central based expression vs. one just for a single pattern expression

### Why?

Currently we have the `ExprRandom` class which is just a single expression pattern `[a] random %*type% [out] of %objects%`
while this probably had plenty of use alone the fact we can do `random %*type%....` should mean `first string out of` should also work but it doesn't. Removal of this Specified name to a general purpose name `ExprClassinfo` is a name that can include more options. 

Whether `random %*type%....` should be made to support an optional type definition is up for debates doing that would mean removal of `random element` makes sense in the `ExprElement` class we'd need to figure out proper syntax for that portion as `random of %objects%` doesn't look good

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this suggestion.

**Comment by ScienceKit** [comment]
This is great in all, but the problem with most community-driven frameworks and languages is legacy code, if the syntax changes, servers that update will face problems with their skripts and old skritps from 3 years ago that still use the `{lore.um}` syntax are like a plague on spigotMC. Dont make another one.

**Reply by Fusezion** [reply]
1. There is no breaking change in this 
2. Using skripts from 3 years ago as a reason is terrible
